src/components/ha-camera-stream.ts

```diff
@@ -176,7 +176,11 @@ class HaCameraStream extends LitElement {
     Hls: HLSModule,
     url: string
   ) {
-    const hls = new Hls();
+    var config = {
+        liveSyncDurationCount: 1,
+        liveBackBufferLength: 60
+    };
```

Comment:
Maybe also add `liveDurationInfinity: true`

Reply:
I didn't quite like it because you cannot seek at all which could be useful if you quickly want to check the past minute or so. Also if you pause you have no way to get back to live and you're pretty much forced to continue playback from where you left off.